crates/storage/codecs/src/alloy/transaction/mod.rs

```diff
@@ -14,26 +14,4 @@ mod tests {
     //
     // this check is to ensure we do not inadvertently add too many fields to a struct which would
     // expand the flags field and break backwards compatibility
-
-    #[cfg(feature = "optimism")]
-    use crate::alloy::transaction::optimism::TxDeposit;
-    use crate::alloy::transaction::{
-        eip1559::TxEip1559, eip2930::TxEip2930, eip4844::TxEip4844, eip7702::TxEip7702,
-        legacy::TxLegacy,
-    };
```

Comment:
these imports should stay here, note that they don't have an optimism feature gate